pkg/notes/notes.go

```diff
@@ -252,6 +252,7 @@ func NoteTextFromString(s string) (string, error) {
 		note := strings.ReplaceAll(result["note"], "#", "&#35;")
 		note = strings.ReplaceAll(note, "\r", "")
 		note = stripActionRequired(note)
+		note = strings.TrimSpace(note)
```

Comment:
I see, this is now part of a dedicated PR to speed-up things: https://github.com/kubernetes/release/pull/870

Reply:
I see that - I'll rebase and pull in those commits here.